### PR TITLE
New package: GoodGameModsLLC.Minion version 3.0.5

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/138755)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 

(Automated response - build 720.)

**Comment by Barradel** [comment]
> Hello @Barradel,
> 
> During installation testing, this application failed to install without user input. Did you forget to add Silent or SilentWithProgress switches?
> 
> This can also happen when a dependency is missing. You can test with https://github.com/microsoft/winget-pkgs#test-your-manifest.
> 
> Please investigate a fix and resubmit the Pull Request.
> 
> You may also try using the [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create) to determine the proper installer type. If the installer type is MSIX, MSI, or a known installer technology like NullSoft, Inno, etc. the wingetcreate tool can detect them and the winget client will know what switches to pass. If it's a .exe installer of an unknown type, you will need to search to determine the proper switches for Silent and SilentWithProgress.
> 
> Template: msftbot/validationError/installation/unattended

No, I did not forget. Put the commands: /SP- /VERYSILENT /NORESTART. I can install silently in my desktop using these commands.

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/g/GoodGameModsLLC/Minion/3.0.5/GoodGameModsLLC.Minion.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: GoodGameModsLLC.Minion
+PackageVersion: 3.0.5
```
[the remark this answers is not in the document]
Comment (Barradel) [reply manifests/g/GoodGameModsLLC/Minion/3.0.5/GoodGameModsLLC.Minion.installer.yaml:5]: Unfortunately the right version is 3.0.5 besides it shows only 3.0. There will be more versions of 3.0.

File: manifests/g/GoodGameModsLLC/Minion/3.0.5/GoodGameModsLLC.Minion.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: GoodGameModsLLC.Minion
+PackageVersion: 3.0.5
+PackageLocale: en-US
+Publisher: Good Game Mods LLC
+PackageName: Minion
+License: Minion EULA
+ShortDescription: Minion Setup
```
[the remark this answers is not in the document]
Comment (Barradel) [reply manifests/g/GoodGameModsLLC/Minion/3.0.5/GoodGameModsLLC.Minion.locale.en-US.yaml]: Done;

File: manifests/g/GoodGameModsLLC/Minion/3.0.5/GoodGameModsLLC.Minion.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: GoodGameModsLLC.Minion
+PackageVersion: 3.0.5
```
[the remark this answers is not in the document]
Comment (Barradel) [reply manifests/g/GoodGameModsLLC/Minion/3.0.5/GoodGameModsLLC.Minion.yaml:5]: Unfortunately the right version is 3.0.5 besides it shows only 3.0. There will be more versions of 3.0.

**Comment by stephengillie** [comment]
Hi @Barradel,

The installation isn't unattended. Is there an installer switch to have the package install silently?

(Automated response - build 729.)

**Comment by Barradel** [comment]
> Hi @Barradel,
> 
> The installation isn't unattended. Is there an installer switch to have the package install silently?
> 
> (Automated response - build 729.)

Hi! Yes, there is. Put the commands: /SP- /VERYSILENT /NORESTART. I can install silently in my desktop using these commands. I already put these, another contributor too. I don't know what's going on. Can you help?

**Comment by stephengillie** [comment]
It appears that the `/SP-` is being stripped from the installer switches between manifest and installer run: 
image

I tried surrounding the line in quotes, but this had no impact.

```PowerShell
InstallerSwitches:
  Silent: "/SP- /VERYSILENT /NORESTART"
  SilentWithProgres: "/SP- /SILENT /NORESTART"
```

**Pull request closed without merging**